[2025-07-25T06:31:18.348+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: etl_pipeline.fetch_infectious_data manual__2025-07-25T06:31:16.855633+00:00 [queued]>
[2025-07-25T06:31:18.360+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: etl_pipeline.fetch_infectious_data manual__2025-07-25T06:31:16.855633+00:00 [queued]>
[2025-07-25T06:31:18.361+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-07-25T06:31:18.381+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): fetch_infectious_data> on 2025-07-25 06:31:16.855633+00:00
[2025-07-25T06:31:18.388+0000] {standard_task_runner.py:60} INFO - Started process 198 to run task
[2025-07-25T06:31:18.394+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'etl_pipeline', 'fetch_infectious_data', 'manual__2025-07-25T06:31:16.855633+00:00', '--job-id', '18', '--raw', '--subdir', 'DAGS_FOLDER/main_pipeline.py', '--cfg-path', '/tmp/tmpa92mhlh4']
[2025-07-25T06:31:18.397+0000] {standard_task_runner.py:88} INFO - Job 18: Subtask fetch_infectious_data
[2025-07-25T06:31:18.496+0000] {task_command.py:423} INFO - Running <TaskInstance: etl_pipeline.fetch_infectious_data manual__2025-07-25T06:31:16.855633+00:00 [running]> on host 37aaa8a9abdb
[2025-07-25T06:31:18.613+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='ingrid' AIRFLOW_CTX_DAG_ID='etl_pipeline' AIRFLOW_CTX_TASK_ID='fetch_infectious_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-25T06:31:16.855633+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-25T06:31:16.855633+00:00'
[2025-07-25T06:31:19.408+0000] {python.py:201} INFO - Done. Returned value was: None
[2025-07-25T06:31:19.418+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=etl_pipeline, task_id=fetch_infectious_data, execution_date=20250725T063116, start_date=20250725T063118, end_date=20250725T063119
[2025-07-25T06:31:19.453+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-07-25T06:31:19.477+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
